src/main/java/hudson/plugins/jira/JiraChangeLogAnnotator.java

```diff
@@ -127,7 +151,7 @@ private void saveIssues(AbstractBuild<?, ?> build, JiraBuildAction a,
         }
     }
 
-    JiraSite getSiteForProject(AbstractProject<?, ?> project) {
+    JiraSite getSiteForProject(Job<?, ?> project) {
```

Comment:
This method i currently used only in same class and with job, i can change it in future if some relevant issue exists.